services/jira/jira-issue.service.js

```diff
@@ -19,7 +19,17 @@ module.exports = class JiraIssue extends LegacyService {
     return [
       {
         title: 'JIRA issue',
-        previewUrl: 'https/issues.apache.org/jira/KAFKA-2896',
+        pattern: ':protocol/:hostAndPath*/:issueKey',
```

Comment:
This probably doesn't matter much as it's the example pattern, however wouldn't we be expecting `:hostAndPath+` rather than `:hostAndPath*`?